src/Data/Parameterized/Context/Safe.hs

```diff
@@ -388,7 +388,7 @@ forIndexRangeImpl i (SizeSucc sz) d f r =
 
 -- | Given an index @i@, size @n@, a function @f@, value @v@, and a
 -- function @f@, the expression @forIndexRange i n f v@ is equivalent
```

Comment:
This isn't your fault, but I noticed that this redundantly mentions "`a function @f@`" a second time. Let's delete the second one.